src/data/cars.js

```diff
@@ -0,0 +1,12 @@
+export const VALID_CARS_RESPONSE_BODY = {

```

Comment:
In fact it is a fixture for particular test. Better move it to fixtures folder where you have your test